docs/03-dev-howtos/18-working-with-emails.md

```diff
@@ -0,0 +1,91 @@
+# Working with emails
+
+
+## Sign up forms in iframes
+
+The email sign up forms include the Guardian Today footer, and the iframe embeds used by editorial in Composer. They are also used in the email sign up interactive atoms, examples [here](https://www.theguardian.com/info/ng-interactive/2016/dec/07/sign-up-for-the-flyer) and [here](https://www.theguardian.com/info/ng-interactive/2016/dec/05/sign-up-for-weekend-reading).
+
+They live in routes like this: `www.theguardian.com/email/form/<type>/<listID>`
+
+The form **type** can be article, footer, plain, plaintone, or plaindark:
+- [article](https://www.theguardian.com/email/form/article/37) >> Commonly used as embed in articles (example here). Button defaults to blue but tone overrides can be applied.
+- [footer](https://www.theguardian.com/email/form/footer/37) >> This is the one used in the footer on every page.
+- [plain](https://www.theguardian.com/email/form/plain/37) >> No header or description. Button will always be guardian-brand blue
+- [plaintone](https://www.theguardian.com/email/form/plaintone/3743) >> No header or description. Submit button splits from input on widescreen. Button defaults to blue but tone overrides can be applied. Used in [email interactives](https://www.theguardian.com/info/ng-interactive/2016/dec/07/sign-up-for-the-flyer).
+- [plaindark](https://www.theguardian.com/email/form/plaindark/3745) Used by [Documentaries](https://www.theguardian.com/world/ng-interactive/2016/oct/14/desert-fire-the-world-cup-rebels-of-kurdistan-video). Depreciated in favour of plaintone (which has a [dark option](http://localhost:9000/email/form/plaintone/3745)) and potentially removable, however, only if all usages can be found.
```

Comment:
Cheers, probably, should have run this through a spell checker.. will do that now!